debian/stretch/foreman/foreman.install

```diff
@@ -26,3 +27,6 @@ bundler.d/facter.rb                      usr/share/foreman/bundler.d
 bundler.d/jsonp.rb                       usr/share/foreman/bundler.d
 public                                   var/lib/foreman
 db                                       var/lib/foreman
+debian/dynflowd.default                  etc/default
+debian/dynflowd.init                     etc/init.d
+debian/dynflowd.service                  lib/systemd/system
```

Comment:
this should be handled by `dh_installinit`

Reply:
Sorry, not an expert, could you explain how? Overriding it in `rules`?

Reply:
a bit, yeah. see `dh_installinit(1)`:

       --name=name
           Install the init script (and default file) as well as upstart job file using the filename name instead of the default filename, which is the package name. When this parameter is used, dh_installinit looks for and installs
           files named debian/package.name.init, debian/package.name.default and debian/package.name.upstart instead of the usual debian/package.init, debian/package.default and debian/package.upstart.

so calling these files `foreman.dynflow.*` and adding the following to `debian/rules` *should* work:

    override_dh_installinit:
        dh_installinit --name=dynflow

(that is, if we only ship one init script)

Reply:
Is there an equivalent thing for systemd? This is just for init.d/default